tests/conftest.py

```diff
@@ -18,3 +24,104 @@ def app():
 @pytest.fixture
 def client(app):
     return app.test_client()
+
+
+@pytest.fixture
+def sql_engine(app):
+    return create_mysql_connection_engine(app.config["RO_CONN_STRING"], app.config["DATABASE_NAME"])
+
+
+@pytest.fixture
+def runs(app, sql_engine):
+    try:
```

Comment:
Can this pattern be refactored out into a function?

Reply:
Could refactor some more, hopefully its a bit cleaner then what is was